app/models/user.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  rolify
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  mount_uploader :avatar, AvatarUploader
+
+  has_many :orders, dependent: :destroy
+
+  validates_presence_of :name
+  validates_presence_of :email
```

Comment:
`validates :name, :email, presence: true` - not sure if that's a working code, but use smth like that when you validate the same thing.